CMakeLists.txt

```diff
@@ -1,23 +1,65 @@
 cmake_minimum_required(VERSION 3.12) #Python3 package locator only added in 3.12
 
+project(cmakedoc DESCRIPTION "CMake automatic documentation generator" LANGUAGES NONE) #Not using any compiled languages so don't list, removes check for compilers
+
 find_package(Python3 COMPONENTS Interpreter)
```

Comment:
you can add `REQUIRED` to `find_package` to get CMake to crash with an informative message that Python3 wasn't found. This also saves you from having to do the `if(Python3_FOUND)` statement later

Reply:
Wasn't quite sure how that worked with external modules, now I understand it though, so it's fixed now.